File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PatientsCreatePatientsRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PatientsCreatePatientsRequestItem {
    @JsonProperty("ActualDate")
    public String actualDate;
    @JsonProperty("ConcentrateOuncesAllowed")
    public Integer concentrateOuncesAllowed;
    @JsonProperty("FlowerOuncesAllowed")
    public Integer flowerOuncesAllowed;
    @JsonProperty("HasSalesLimitExemption")
    public Boolean hasSalesLimitExemption;
    @JsonProperty("InfusedOuncesAllowed")
    public Integer infusedOuncesAllowed;
    @JsonProperty("LicenseEffectiveEndDate")
    public String licenseEffectiveEndDate;
    @JsonProperty("LicenseEffectiveStartDate")
    public String licenseEffectiveStartDate;
    @JsonProperty("LicenseNumber")
    public String licenseNumber;
    @JsonProperty("MaxConcentrateThcPercentAllowed")
    public Integer maxConcentrateThcPercentAllowed;
    @JsonProperty("MaxFlowerThcPercentAllowed")
    public Integer maxFlowerThcPercentAllowed;
    @JsonProperty("RecommendedPlants")
    public Integer recommendedPlants;
    @JsonProperty("RecommendedSmokableQuantity")
    public Double recommendedSmokableQuantity;
    @JsonProperty("ThcOuncesAllowed")
    public Integer thcOuncesAllowed;
}

File: sdks/thunkmetrc-rs/wrapper/src/models/patients_create_patients_request_item.rs
use serde::{Deserialize, Serialize};
#[allow(unused_imports)]
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PatientsCreatePatientsRequestItem {
    #[serde(rename = "actual_date", skip_serializing_if = "Option::is_none")]
    pub actual_date: Option<String>,
    #[serde(rename = "concentrate_ounces_allowed", skip_serializing_if = "Option::is_none")]
    pub concentrate_ounces_allowed: Option<i64>,
    #[serde(rename = "flower_ounces_allowed", skip_serializing_if = "Option::is_none")]
    pub flower_ounces_allowed: Option<i64>,
    #[serde(rename = "has_sales_limit_exemption", skip_serializing_if = "Option::is_none")]
    pub has_sales_limit_exemption: Option<bool>,
    #[serde(rename = "infused_ounces_allowed", skip_serializing_if = "Option::is_none")]
    pub infused_ounces_allowed: Option<i64>,
    #[serde(rename = "license_effective_end_date", skip_serializing_if = "Option::is_none")]
    pub license_effective_end_date: Option<String>,
    #[serde(rename = "license_effective_start_date", skip_serializing_if = "Option::is_none")]
    pub license_effective_start_date: Option<String>,
    #[serde(rename = "license_number", skip_serializing_if = "Option::is_none")]
    pub license_number: Option<String>,
    #[serde(rename = "max_concentrate_thc_percent_allowed", skip_serializing_if = "Option::is_none")]
    pub max_concentrate_thc_percent_allowed: Option<i64>,
    #[serde(rename = "max_flower_thc_percent_allowed", skip_serializing_if = "Option::is_none")]
    pub max_flower_thc_percent_allowed: Option<i64>,
    #[serde(rename = "recommended_plants", skip_serializing_if = "Option::is_none")]
    pub recommended_plants: Option<i64>,
    #[serde(rename = "recommended_smokable_quantity", skip_serializing_if = "Option::is_none")]
    pub recommended_smokable_quantity: Option<i64>,
    #[serde(rename = "thc_ounces_allowed", skip_serializing_if = "Option::is_none")]
    pub thc_ounces_allowed: Option<i64>,
}
